extensions/wikia/AdEngine/js/video/player/porvata/porvata.js

```diff
@@ -96,6 +97,15 @@ define('ext.wikia.adEngine.video.player.porvata', [
 					params.onReady(video);
 				}
 
+				if (mercuryListener) {
```

Comment:
I'm not sure if Porvata should be aware of if it's running on mercury or not. How about doing it other way around, for instance trigger event `allAdsCompleted` from some `mercury-related` file on page change? Or create custom event like `onPlayerDestroy`and dispatch this event on player from some `mercury-related` file

/cc @fraszczakszymon